src/main/scala/tech/cryptonomic/conseil/util/RouteUtil.scala

```diff
@@ -0,0 +1,106 @@
+package tech.cryptonomic.conseil.util
+
+import java.util.UUID
+
+import akka.http.scaladsl.model.StatusCodes.InternalServerError
+import akka.http.scaladsl.model.{HttpEntity, HttpResponse, RemoteAddress, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.directives.BasicDirectives
+import akka.http.scaladsl.server.{Directive, ExceptionHandler, RequestContext, Route}
+import akka.stream.Materializer
+import cats.effect.concurrent.MVar
+import cats.effect.{Concurrent, IO}
+import com.typesafe.scalalogging.LazyLogging
+import org.slf4j.MDC
+
+class RouteUtil(implicit concurrent: Concurrent[IO]) extends LazyLogging {
```

Comment:
Please document the purpose of this file.

Reply:
Sure, added

Reply:
ok, renamed